components/WalletModal/index.js

```diff
@@ -10,7 +10,7 @@ import WalletCard from 'components/WalletModal/WalletCard'
 import { walletlink, injected } from 'utils/constants/connectors'
 import ANIMATIONS from 'utils/constants/animate-icons'
 
-import { AnalyticActions, AnalyticCategories, createEvent, analytics } from "utils/analytics";
+import { AnalyticActions, AnalyticCategories, createEvent, useAnalytics } from "contexts/analytics";
```

Comment:
Not sure if this is the cause, but I was getting an error when connecting initially and the preview is not acting like it is not on Avalanche NEtwork. Could be something with the preview generation though. I have to dig into it.

![image](https://user-images.githubusercontent.com/3376497/176101327-a5d59ae9-a0c5-4fa9-ab51-71b2053b3881.png)